src/actions/authActions.ts

```diff
@@ -101,7 +102,7 @@ export const verifyToken = (dispatch) => async () => {
           resolve(data);
           return;
         }
-        const userData = tokenGetClaims(token);
+        const userData: {[key: string]: any} = tokenGetClaims(token);
```

Comment:
we should define a more defined object type for userData, store it in a types folder or smth so it can be reused throughout the website